dataactbroker/handlers/accountHandler.py

```diff
@@ -717,4 +720,39 @@ def setSkipGuide(self, session):
             exc = ResponseException("skip_guide must be a boolean", StatusCode.CLIENT_ERROR)
             return JsonResponse.error(exc, exc.status)
         userDb.session.commit()
-        return JsonResponse.create(StatusCode.OK,{"message":"skip_guide set successfully","skip_guide":skipGuide})
\ No newline at end of file
+        return JsonResponse.create(StatusCode.OK,{"message":"skip_guide set successfully","skip_guide":skipGuide})
+
+    def emailUsers(self, system_email, session):
+        """ Send email notification to list of users """
+        requestDict = RequestDictionary(self.request)
+        if not (requestDict.exists("users") and requestDict.exists("submission_id") and requestDict.exists("email_template")):
+            exc = ResponseException("Email users route requires users, email_template, and submission_id", StatusCode.CLIENT_ERROR)
+            return JsonResponse.error(exc, exc.status)
+
+        uid = session["name"]
+        current_user = self.interfaces.userDb.getUserByUID(uid)
+
+        user_ids = requestDict.getValue("users")
+        submission_id = requestDict.getValue("submission_id")
+        # Check if submission id is valid
+        self.jobManager.getSubmissionById(submission_id)
+
+        template_type = requestDict.getValue("email_template")
+        # Check if email template type is valid
+        self.userManager.getEmailTemplate(template_type)
+
+        users = []
+
+        link = "".join([AccountHandler.FRONT_END, '#/reviewData/', str(submission_id)])
+        emailTemplate = {'[REV_USER_NAME]': current_user.name, '[REV_URL]': link}
+
+        for user_id in user_ids:
+            # Check if user id is valid, if so add User object to array
+            users.append(self.userManager.getUserByUID(user_id))
```

Comment:
Confirmed with @kevinli-work that the front-end is populated via the `list_users` route, which only returns the appropriate users. Thanks!